ruby/ql/test/library-tests/dataflow/local/local_dataflow.rb

```diff
@@ -85,6 +85,7 @@ def test_case x
   in { a: f } then sink(f) # $ hasTaintFlow=1
   in { foo: 1, g: } then sink(g) # $ hasTaintFlow=1
   in { x: } then sink(x) # $ hasTaintFlow=1
+  sink(z) # $ hasTaintFlow=1
```

Comment:
Remove `# $ hasTaintFlow=1`. However, it would be nice with a real test for flow out of a `case` branch into the result of the `case` expression (if we don't already have that).

Reply:
The `sink(z)` is actually on the wrong line, it should be placed after the `end` that closes the `case`.

Reply:
Looks like the branches of the `case` need indentation. I don't understand why there is flow from `source(1)` to `z` in this case?

Reply:
I added `; x` in `in { x: } then sink(x); x`

Reply:
Do you understand why there was flow prior to adding `; x`?

Reply:
There wasn't any flow prior to adding `; x`

Reply:
Ah, good.